sandbox/registration-processor-dmz.properties

```diff
@@ -164,6 +164,10 @@ registration.processor.max.retry=3
 # for search API response consumed by regclient
 mosip.registration.processor.registration.status.external-statuses-to-consider-processed=UIN_GENERATED,REREGISTER,REJECTED,REPROCESS_FAILED	
 
+mosip.registration.processor.lostrid.version=1.0
+mosip.registration.processor.postalcode.req.url=https://dev.mosip.net/v1/masterdata/registrationcenters
+mosip.registration.processor.lostrid.id=mosip.registration.lostrid
```

Comment:
Move this next line to mosip.registration.processor.registration.sync.id

Reply:
taken care